nucliadb_utils/nucliadb_utils/storages/local.py

```diff
@@ -57,7 +57,7 @@ async def move(
         destination_bucket_path = self.storage.get_bucket_path(destination_bucket_name)
         origin_path = f"{origin_bucket_path}/{origin_uri}"
         destination_path = f"{destination_bucket_path}/{destination_uri}"
-        os.rename(origin_path, destination_path)
+        os.renames(origin_path, destination_path)
```

Comment:
`os.renames` also deletes the parent of the `origin_path` as a side effect if it's empty. Not sure we want that. Maybe better to just create the dirs manually:
```python
parent = os.path.split(origin_path)[0]
os.makedirs(parent, exist_ok=True)
```